src/Application.vue

```diff
@@ -157,6 +158,12 @@ export default {
         setThemeInstance(theme.instances[0])
         setActiveTheme(theme)
 
+        // There are old settings apply them to the new theme
```

Comment:
how are you handling failures if the network request fails?

Reply:
@squallstar There is a `catch()` where I do the request and the error is presented in the UI.
Also if that fails everything else stops, so therefore the user will need to try again.